src/node.h

```diff
@@ -209,8 +210,26 @@ NODE_EXTERN void Init(int* argc,
 class IsolateData;
 class Environment;
 
-NODE_EXTERN IsolateData* CreateIsolateData(v8::Isolate* isolate,
-                                           struct uv_loop_s* loop);
+class WorkerSupportingPlatform : public v8::Platform {
```

Comment:
Can the name be made more generic, like maybe `MultiIsolatePlatform`? After all, this is a public API.